docs/cog_guides/alias.rst

```diff
@@ -0,0 +1,270 @@
+.. _alias:
+
+=====
+Alias
+=====
+
+This is the cog guide for the alias cog. You will
+find detailed docs about the usage and the commands.
+
+``[p]`` is considered as your prefix.
+
+.. note:: To use this cog, load it by typing this::
+
+        [p]load alias
+
+.. _alias-usage:
+
+-----
+Usage
+-----
+
+This cog allows to create shortcuts for commands.
+
+You can do things like this:
+
+.. code-block:: none
+
+    [p]cleanup messages 42
+    equals to
+    [p]clean 42
+
+or even this:
+
+.. code-block:: none
+
+    [p]search sc Au temps en emporte le Z event
+    equals to
+    [p]soundcloud Au temps en emporte le Z event
+
+In the first example, we made an alias named ``clean`` that will
+invoke the ``cleanup messages`` command.
+
+In the second example, we made an alias called ``soundcloud`` that will
+invoke the ``search sc`` command (search a song to play on SoundCloud. As you
+can see, you can also add arguments to your alias. Then you can add the
```

Comment:
Like I said in a comment above, maybe make the first example something simple like `[p]play` -> `[p]p` and explain that example, *then* introduce an example (I personally think `cleanup messages` is better) with parameters and include this line.